pep-3135.txt

```diff
@@ -197,41 +200,33 @@ concerns are obvious. Guido has suggested that this should be excluded from
 this PEP on the principle of KISS (Keep It Simple Stupid).
 
 
-
 History
 =======
-12-Mar-2009 - Updated to reflect the current state of implementation.
+12-Mar-2009
+  - Updated to reflect the current state of implementation.
+
+29-Apr-2007
```

Comment:
These timestamps are also out of order, let's have them either ascending or descending.